### PR TITLE
feat: configurable journey planner api configurations

### PR DESCRIPTION
**Pull request by mortennordseth**
Because the journey planner API configurations differ for AtB, NFK, and FRAM we need to have them configurable. This makes that possible by using the org data.

Closes https://github.com/AtB-AS/kundevendt/issues/16416

### CODE REVIEW

**Comment by adriansberg** [comment]
How is this configured for the app? In the bff?

**Review by mikaelbr — CHANGES_REQUESTED** [review]
I think this should be part of org data instead. It isn't needed as a secret but just easily accessible information

**Comment by mortennordseth** [comment]
I've moved the configuration to the org data and updated the values to match what is used in the app. 

> How is this configured for the app? In the bff?

This is configured in the app. Some are editable with preferences

File: orgs/schema-validation.json [hunk 1]
```diff
@@ -75,6 +75,27 @@
             "supportUrl"
           ],
           "type": "object"
+        },
+        "journeyAPIConfigurations": {
```
Comment (adriansberg) [inline orgs/schema-validation.json:79]:
```suggestion
        "journeyApiConfigurations": {
```

File: orgs/schema-validation.json [hunk 2]
```diff
@@ -75,6 +75,27 @@
...
+          "additionalProperties": false,
+          "properties": {
+            "WAIT_RELUCTANCE": {
```
Comment (adriansberg) [inline orgs/schema-validation.json:82]: Do these need to be capitalised when they're not defined as constants, but as properties in an object?

**Review by adriansberg — APPROVED** [review]
Looks good, other than some nitpicky comments.

**Pull request merged**